nextbike/prediction/math_prepare_feature.py

```diff
@@ -21,19 +21,34 @@ def create_dummies(p_df):
     return df_dummies
 
 
-# TODO: Add docstring
 def create_new_features(p_X):
-    X = p_X
-    X["hour_squared"] = np.square(p_X["hour_start"])
-    X["day_squared"] = np.square(p_X["day_start"])
-    X["month_squared"] = np.square(p_X["month_start"])
-    X["weekend_squared"] = np.square(p_X["Weekend"])
+    """Create new features which are usefull for prediction performance.
+
+    Example methods for feature engineering could be found here:
+    https://de.devoteam.com/blog-post/bedeutung-der-feature-engineering-methoden-2/
+    Args:
+        p_X (DataFrame): Dataframe of existing features (matrix)
+    Returns:
+        p_X (DataFrame): Dataframe with existing and new added features (matrix)
+    """
+    p_X["hour_squared"] = np.square(p_X["hour_start"])
+    p_X["day_squared"] = np.square(p_X["day_start"])
+    p_X["month_squared"] = np.square(p_X["month_start"])
+    p_X["weekend_squared"] = np.square(p_X["Weekend"])
     # X["day_squared"] = p_X["day_start"]/p_X["p_bikes_start"]
```

Comment:
X["day_squared"] = p_X["day_start"]/p_X["p_bikes_start"]
should be changed to:
**p_X**["day_squared"] = p_X["day_start"]/p_X["p_bikes_start"]

Reply:
Comment is deleted in next merge